src/componentsBasic/home/homeSections/homeSubscribe/HomeSubscribe.js

```diff
@@ -1,8 +1,14 @@
+import React, { useState } from "react";
+
```

Comment:
GENERAL FIXES:
1. All components should have name as UpperCase. button.js - not correct. Button.js - better, Button.jsx the best
2. Rename imports with styles to LowerCase. Because you have file as footer.scss, but import Footer.scss
3. Rename componentsBasic - pages
4. Rename componentsUniversal - components